django/text_extractor/views.py

```diff
@@ -88,26 +87,26 @@ def submit_document(request):
                     if idx > 0:  # Exclude TOC from file names to merge
                         file_names_to_merge.append(file_name)
                 else:
-                    file_id = f"{user_name}_{current_time}_OCR_{input_name}.pdf"
-                    text_id = f"{user_name}_{current_time}_{input_name}.txt"
```

Comment:
Don't need these special IDs, the normal UUID provided by SecureModel will be fine